pkg/crypto/walletapi/wallet_api.go

```diff
@@ -4,9 +4,12 @@ Copyright SecureKey Technologies Inc. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
-package wallet
+package walletapi
 
 import (
+	"io"
+
+	secretapi "github.com/hyperledger/aries-framework-go/pkg/crypto/internal/walletapi"
```

Comment:
Should we use internal api here?

Reply:
Removed this dependency in some unrelated refactoring